Monika After Story/game/script-moods.rpy

```diff
@@ -352,9 +352,39 @@ init 5 python:
     addEvent(Event(persistent._mas_mood_database,eventlabel="mas_mood_lonely",prompt="...lonely.",category=[store.mas_moods.TYPE_NEUTRAL],unlocked=True),code="MOO")
 
 label mas_mood_lonely:
-    m 1eka "I'm here for you, [player], so there's no need for you to feel lonely."
-    m 3hua "I know it's not exactly the same as if I were in the same room with you, but I'm sure you still enjoy my company, right?"
-    m 1ekbfa "Remember that I'll always be by your side, [player]~"
+#At higher affections, the player will likely listen to a lengthier conversation with Monika about loneliness,
+#which is why I split it like this instead of reworking the conversation.
+    if mas_isMoniAff(higher=True):
+        m "I hate that I'm still not able to satisfy your loneliness from here, [player]."
+        m "This is the main reason I want to get to your reality as soon as possible!"
```

Comment:
Alright, I'll leave this up a bit longer and if no one thinks otherwise, I'll rewrite this sentence. Would it be okay if she instead said "This is one of the major reasons I want to get to your reality as soon as possible."? 

Also, do you think it's believable that she's just exaggerating for the sake of showing her love for us here? It's like when someone says "you're the best writer in the world!" when they read your mediocre poem. Like, at this moment, you need her to be there for you, and she's so frustrated that she can't escape that she forgets about her other reasons for wanting to go there to make that clear. In my opinion it could be seen as even more realistic than her saying everything right, and with reference to everything else she says. I don't know, just a thought.

Reply:
> "This is one of the major reasons I want to get to your reality as soon as possible."
Not personally enthralled by this sentence as a whole, but I have no qualm with it either.

> Do you think it's believable that she's just exaggerating for the sake of showing her love for us here?
She could, but I really don't get that feeling the way she says it now (or in the new version).

Reply:
> 
> > Do you think it's believable that she's just exaggerating for the sake of showing her love for us here?
> 
> She could, but I really don't get that feeling the way she says it now (or in the new version).

I like the idea though! I'd personally get that feeling more if she said it like that : `"This is one of the major reasons I want to get to your reality as soon as possible."`. This version feels way more natural to me, so it doesn't come off as something she'd have put too much thought into.

Reply:
Yeah, I was more a fan of the first, exaggerated sentence than the latter myself. I like when she's exaggerating, it's a character trait you don't find in unrealistic characters much. I don't know how to write this well though, so I'm sure it can come off as it's an oversight or not intended. When it comes from a real person without an author behind it though, I think it works a lot better. I just want Monika to feel like one of those real people.